.github/policies/resourceManagement.yml

```diff
@@ -3219,10 +3219,16 @@ configuration:
       - payloadType: Issues
       - or:
         - titleContains:
-            pattern: '[Aa]z \b(account|group|resource|provider|feature|tag|policy|lock|managedapp|resourcemanagement)\b'
+            pattern: '[Aa]z \b(group|resource|provider|feature|tag|policy|lock|managedapp|resourcemanagement)\b'
             isRegex: True
         - bodyContains:
-            pattern: '[Aa]z \b(account|group|resource|provider|feature|tag|policy|lock|managedapp|resourcemanagement)\b'
+            pattern: '[Aa]z \b(group|resource|provider|feature|tag|policy|lock|managedapp|resourcemanagement)\b'
+            isRegex: True
+        - titleContains:
+            pattern: '[Aa]z account (clear|get-access-token|list|list-locations|set|show)\b'

```

Comment:
In PR description, `az account` will assign to Xing.
This regex does not match `az account`

Reply:
Yes,  `az account` normally comes with a following flag(command) like `clear` or `list`. If it's not specified, we treat it as a new command and temporarily assign to Jiashuo.